libs/sdk-ui-dashboard/src/model/commandHandlers/layout/moveSectionItemHandler.ts

```diff
@@ -233,6 +249,17 @@ export function* moveSectionItemHandler(
                       }),
                   ]
                 : []),
+            ...(shouldChangeSize
+                ? [
+                      layoutActions.changeItemWidth({
+                          layoutPath:
+                              targetIndex === undefined
+                                  ? [{ sectionIndex: targetSectionIndex, itemIndex: targetItemIndex }]
+                                  : targetIndex,
+                          width: itemWithNormalizedSize.size.xl.gridWidth,
```

Comment:
Should this be always xl?

Reply:
yes whole edit mode works in XL screen structure, even actual screen is L